<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialPortSelectDialog</class>
 <widget class="QDialog" name="serialPortSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configure LVDP serial port settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>200</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="serialPortGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Select LVDP serial port:</string>
   </property>
   <widget class="QComboBox" name="serialPortSelectComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="staticLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM port:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="serialPortSpeedGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Select LVDP communication speed:</string>
   </property>
   <widget class="QRadioButton" name="bps9600RadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>9600 BPS</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">serialPortSpeedButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="bps2400RadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2400 BPS</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">serialPortSpeedButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="bps1200RadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1200 BPS</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">serialPortSpeedButtonGroup</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>serialPortSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="serialPortSpeedButtonGroup"/>
 </buttongroups>
</ui>
